packages/react-vapor/src/components/textarea/tests/TextArea.spec.tsx

```diff
@@ -93,46 +96,81 @@ describe('TextArea', () => {
             expect(onChangeCallback).toHaveBeenCalledTimes(1);
         });
 
+        it('should contains validation message if changed value is invalid', () => {
+            const validation = (value: string) => value !== '';
+            const validationMessage = 'invalid value';
+            const invalidValue = '';
+            jasmine.clock().install();
+
+            wrapper.setProps({validate: validation, validationMessage: validationMessage, value: invalidValue});
+            act(() => {
+                wrapper.update();
+            });
+            expect(wrapper.contains(validationMessage)).toBeFalsy();
+
+            wrapper.setProps({value: invalidValue});
+            act(() => {
+                wrapper.update();
+            });
+
+            jasmine.clock().tick(301);
+            expect(wrapper.contains(validationMessage)).toBeTruthy();
+        });
+
+        it('should call prop validate on value change', () => {
+            const validate = jasmine.createSpy('validate');
+            hookWrapper = mount(<TextArea id="textarea-id" validate={validate} />);

```

Comment:
Can you use shallow instead? I know the other tests are made with mount but we try to avoid it when possible